src/controller/tests/data_model/DataModelFixtures.cpp

```diff
@@ -146,16 +146,17 @@ static CHIP_ERROR ReadSingleClusterData(const Access::SubjectDescriptor & aSubje
 
             return valueEncoder.Encode(++gInt16uTotalReadCount);
         }
+        // TODO
```

Comment:
TODO what?

Reply:
oops ... hard marker that "this test still needs fixing". Removed.